integration-test/src/test/java/org/sagebionetworks/IT500SynapseJavaClientTeamTest.java

```diff
@@ -650,31 +650,33 @@ public void testMembershipInvitationAPI() throws Exception {
 			// as expected
 		}
 
+		
+		// TODO Commented out the membership invitation by email address, feature disabled because of https://sagebionetworks.jira.com/browse/PLFM-5970
 		// create an invitation with null inviteeId and non null inviteeEmail
-		dto.setInviteeId(null);
-		dto.setInviteeEmail(TEST_EMAIL);
-		MembershipInvitation mis = synapseOne.createMembershipInvitation(dto, MOCK_ACCEPT_INVITATION_ENDPOINT, MOCK_NOTIFICATION_UNSUB_ENDPOINT);
-		InviteeVerificationSignedToken token = synapseTwo.getInviteeVerificationSignedToken(mis.getId());
-		// test if getInviteeVerificationSignedToken succeeded
-		assertNotNull(token);
-		String inviteeId = inviteeUserProfile.getOwnerId();
-		assertEquals(inviteeId, token.getInviteeId());
-		assertEquals(mis.getId(), token.getMembershipInvitationId());
-
-		// update the inviteeIinviteeUserProfile.getOwnerId() of the invitation
-		synapseTwo.updateInviteeId(mis.getId(), token);
-		mis = synapseTwo.getMembershipInvitation(mis.getId());
-		// test if updateInviteeId succeeded
-		assertEquals(inviteeId, mis.getInviteeId());
-
-		// delete the second invitation
-		synapseOne.deleteMembershipInvitation(mis.getId());
-		try {
-			synapseOne.getMembershipInvitation(mis.getId());
-			fail("Failed to delete membership invitation.");
-		} catch (SynapseException e) {
-			// as expected
-		}
+//		dto.setInviteeId(null);
+//		dto.setInviteeEmail(TEST_EMAIL);
+//		MembershipInvitation mis = synapseOne.createMembershipInvitation(dto, MOCK_ACCEPT_INVITATION_ENDPOINT, MOCK_NOTIFICATION_UNSUB_ENDPOINT);
+//		InviteeVerificationSignedToken token = synapseTwo.getInviteeVerificationSignedToken(mis.getId());
+//		// test if getInviteeVerificationSignedToken succeeded
+//		assertNotNull(token);
+//		String inviteeId = inviteeUserProfile.getOwnerId();
+//		assertEquals(inviteeId, token.getInviteeId());
+//		assertEquals(mis.getId(), token.getMembershipInvitationId());
+//
+//		// update the inviteeIinviteeUserProfile.getOwnerId() of the invitation
+//		synapseTwo.updateInviteeId(mis.getId(), token);
+//		mis = synapseTwo.getMembershipInvitation(mis.getId());
+//		// test if updateInviteeId succeeded
+//		assertEquals(inviteeId, mis.getInviteeId());
+//
+//		// delete the second invitation
+//		synapseOne.deleteMembershipInvitation(mis.getId());
+//		try {
+//			synapseOne.getMembershipInvitation(mis.getId());
+//			fail("Failed to delete membership invitation.");
+//		} catch (SynapseException e) {
+//			// as expected
+//		}
```

Comment:
We do not have commented out / dead code in our code base.  The lines should be removed altogether.